src/domain/users/index.types.d.ts

```diff
@@ -58,7 +62,7 @@ type User = {
 
 type NewUserInfo = {
   phone: PhoneNumber
-  phoneMetadata: PhoneMetadata | null
+  phoneMetadata?: PhoneMetadata
```

Comment:
this breaks PhoneMetadataValidator null validation `if (phoneMetadata === null || !phoneMetadata.carrier)`.

Reply:
if we're not catching this in unit tests, maybe it's worth adding an additional test